### PR TITLE
Init: dependencies, theme and graphQL clients

### PR DESCRIPTION
**Pull request by Ilhan-brandverse-dev**
Color pallet is not decided yet.

### CODE REVIEW

File: lib/theme/text_theme.dart [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  displayMedium: TextStyle(
+      fontFamily: 'Poppins',
+      fontSize: 27,
+      fontWeight: FontWeight.w600,
+      color: AppColors.blueGrey),
+  displaySmall: TextStyle(
+    color: AppColors.darkBlue,
+    fontSize: 18,
+    fontWeight: FontWeight.w800,
+    fontFamily: 'Poppins',
```
Comment (mhussain122) [inline lib/theme/text_theme.dart:36]: Avenir Next (Avenir family) is a good font for finance apps, so can we remove Poppins? just a suggestion

File: lib/graphql_clients/exception_parser.dart [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+class ExceptionParser {
+  static graphQLErrorParser(OperationException? error) {
+    if (!isEmptyOrNull(error?.linkException?.originalException.toString()) &&
+        !error!.linkException!.originalException
+            .toString()
+            .contains('SocketException')) {
+      return !isArrayEmptyOrNull(error.graphqlErrors)
+          ? Future.error(error.graphqlErrors.first.message.toString())
+          : Future.error('Something went wrong');
+    }
```
Comment (mhussain122) [inline lib/graphql_clients/exception_parser.dart:13]: Can we implement translations from the beginning too?

File: lib/theme/app_bar_theme.dart [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+import 'colors/grey_pallete.dart';
```
Comment (mhussain122) [inline lib/theme/app_bar_theme.dart:3]: package import

File: lib/theme/bottom_nav_theme.dart [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+
+import 'colors/app_colors.dart';
+
```
Comment (mhussain122) [inline lib/theme/bottom_nav_theme.dart:4]: package import

File: lib/theme/colors/app_colors.dart [hunk 5]
```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+
+class AppColors {
+  static const Color primaryOrange = Color.fromRGBO(254, 69, 88, 1);
+  static const Color dimOrange = Color(0xFFFFEDE4);
+  static const Color navBarOrange = Color(0xFFEE3D50);
+  static const Color orangeTint = Color(0xFFFFEDE4);
```
Comment (mhussain122) [inline lib/theme/colors/app_colors.dart:7]: orange colors?

File: lib/theme/colors/app_colors.dart [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  static const Color white = Color(0xFFFFFFFF);
+  static const Color black = Color(0xFF000000);
+  static const Color black26 = Color(0x42000000);
+  static const Color errorColor = Color(0xFFF03333);
+  static const Color successColor = Color(0xFF0FC982);
+  static Color transparent = Colors.transparent;
+
+  static const MaterialColor customOrange =
+      MaterialColor(_mcgpalette0PrimaryValue, <int, Color>{
+    50: Color.fromRGBO(254, 69, 88, .1),
```
Comment (mhussain122) [inline lib/theme/colors/app_colors.dart:38]: orange colors?

File: lib/graphql_clients/exception_parser.dart (hunk 2 above)
Comment (Ilhan-brandverse-dev) [reply lib/graphql_clients/exception_parser.dart:13]: Translations will not be the part of this project

File: lib/theme/theme_data.dart [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+import 'bottom_nav_theme.dart';
+import 'card_theme.dart';
+import 'colors/app_colors.dart';
+import 'colors/grey_pallete.dart';
+
+ThemeData theme = ThemeData(
+    // fontFamily: '',
+    primarySwatch: AppColors.customOrange,
+    appBarTheme: appBarTheme,
```
Comment (mhussain122) [inline lib/theme/theme_data.dart:13]: assign default font family?

File: lib/theme/colors/app_colors.dart (hunk 6 above)
Comment (Ilhan-brandverse-dev) [reply lib/theme/colors/app_colors.dart:38]: A comment is already added in PR description regarding colors.
Please read it

File: lib/theme/text_theme.dart (hunk 1 above)
Comment (Ilhan-brandverse-dev) [reply lib/theme/text_theme.dart:36]: Will discuss

**Comment by mhussain122** [comment]
Use AppTheme.loadTheme from design system instead?

**Review by mhussain122 — APPROVED** [review]

**Review by iffatak — APPROVED** [review]

**Pull request merged**